File: src/SOP8/Base/History/Base.History.IBLL/Response/ResponseSensorDetectHistory.cs
using System.Collections.Generic;
using Response;

namespace Base.History.IBLL.Response
{
    using Models.History;

    public class ResponseSensorDetectHistory : PagingMessageResult
    {
        private List<SensorDetectHistory> m_histories = new List<SensorDetectHistory>();

        public List<SensorDetectHistory> Histories
        {
            get { return m_histories; }
        }

        public ResponseSensorDetectHistory()
            : base()
        {
        }

        public ResponseSensorDetectHistory(bool success, string message)
            : base(success, message)
        {
        }
    }
}

File: src/SOP8/Egovframework/Base/base-history-ibll/src/main/java/egovframework/base/history/ibll/response/ResponseSensorDetectHistory.java
package egovframework.base.history.ibll.response;

import egovframework.base.history.ibll.models.history.SensorDetectHistory;
import egovframework.base.response.PagingMessageResult;

import java.util.ArrayList;
import java.util.List;

public class ResponseSensorDetectHistory extends PagingMessageResult {

    private List<SensorDetectHistory> histories = new ArrayList<>();

    // Constructors
    public ResponseSensorDetectHistory() {
        super();
    }

    public ResponseSensorDetectHistory(boolean success, String message) {
        super(success, message);
    }

    // Getter and Setter
    public List<SensorDetectHistory> getHistories() {
        return histories;
    }

    public void setHistories(List<SensorDetectHistory> histories) {
        this.histories = histories;
    }
}